### DISCUSSION TITLE
Measurement Channel renaming

### DISCUSSION THREAD

**Discussion by gual11** [danielhrisca/asammdf#862] [Q&A]
Hi, 
is it possible to change the channel names in a mf4 file?

**Comment by danielhrisca** [comment]
There is no straightforward way to edit the channel name inside the file

**Reply by gual11** [reply]
okay so not possible at all? means after creating an mf4 file these names are fix

**Reply by danielhrisca** [reply]
```python
mdf = MDF("my_file.mf4")
ch = mdf.groups[2].channels[3]
ch.name = "NewName"
mdf.save("with_new_name.mf4")
```
